src/ast/mod.rs

```diff
@@ -646,12 +646,16 @@ pub enum Expr {
         regexp: bool,
     },
     /// `ANY` operation e.g. `foo > ANY(bar)`, comparison operator is one of `[=, >, <, =>, =<, !=]`
+    /// https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#_8_9_quantified_comparison_predicate
```

Comment:
I'm thinking we can link to [snowflake](https://docs.snowflake.com/en/sql-reference/operators-subquery#all-any) instead? Since that's a supported dialect that has the subquery functionality. The current link  doesnt seem to refer to any specific dialect so it could add confusion later on when trying to figure what features in the parser to maintain